### ISSUE TITLE
Support Plone 5.2

### ISSUE THREAD

**Issue by ewohnlich** [castlecms/wildcard.hps#1]
From my testing it looks like the only thing necessary to support Plone 5.2 (and probably 6) is to update a couple of the imports, e.g. "Products.ZCatalog.Lazy" to "ZTUtils.Lazy". Is there interest in keeping this updated with newer versions of Plone?

**Comment by zombified** [comment]
Yes, currently we need it to maintain support for 5.0, but will definitely need to get it to be compatible with 5.2 and 6

**Comment by ewohnlich** [comment]
I've made some minimal changes to a fork here https://github.com/imsweb/wildcard.hps/tree/plone6 and have this working on 5.2 (production) and 6.0 (dev).

**Comment by zombified** [comment]
excellent, thank you! we'll take a look at it
